packages/calcite-components/src/components/tree-item/tree-item.tsx

```diff
@@ -227,17 +231,27 @@ export class TreeItem implements ConditionalSlotComponent, InteractiveComponent
 
     const checkbox =
       this.selectionMode === "ancestors" ? (
-        <label class={CSS.checkboxLabel} key="checkbox-label">
-          <calcite-checkbox
-            checked={this.selected}
+        <div class={CSS.checkboxContainer}>
+          <div
+            aria-checked={toAriaBoolean(this.selected)}
```

Comment:
I think this `div` might need a role (maybe checkbox). I ran the test app through aXe toolbar and it showed this:

<img width="720" alt="image" src="https://github.com/user-attachments/assets/04f7da36-b47a-49c6-bf0f-fb827693f0d4">

@geospatialem do you know what this might need?

I'm not sure we even need aria-checked here because the tree-item role has aria-selected so that should really account for selection to a screen reader right?

Reply:
@driskull Adding `role="checkbox"` gets rid of the error above. Do we want to get rid of `aria-checked` entirely?

cc @geospatialem

Reply:
From testing, it looks like we should omit this `div` to only contain the `class` and `tabIndex`. For instance:

```tsx
<div class={CSS.checkboxContainer}>
  <div class={CSS.checkbox} tabIndex={-1}>
    <calcite-icon
      ...
```

Reply:
Instead we'd add the contents to the `Host` `tree-item` element to announce the selection via `aria-live="polite"`:

```tsx
<Host
  aria-checked={this.selectionMode === "ancestors" ? toAriaBoolean(this.selected) : undefined}
  aria-expanded={this.hasChildren ? toAriaBoolean(isExpanded) : undefined}
  aria-hidden={toAriaBoolean(hidden)}
  aria-live="polite"
  calcite-hydrated-hidden={hidden}
  role="treeitem"
  tabIndex={this.disabled ? -1 : 0}
>
```

The above would also remove the `aria-selected` attribute, which currently contains:

```tsx
  ...
  aria-selected={this.selected ? "true" : showCheckmark ? "false" : undefined}
  ...
```

Not sure if there is an impact there, but I didn't notice one in testing.

The above two code changes allow JAWS and NVDA to hear the selections and live changes, in addition to passing all axe conditions.